docs/core/linux-prerequisites.md

```diff
@@ -22,25 +22,26 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 .NET Core 2.x is supported on the following Linux distributions/versions:
 
- * Red Hat Enterprise Linux 7 x64
- * CentOS 7 x64
- * Oracle Linux 7 x64
- * Fedora 25, 26 x64
- * Debian 9, 8.7+ x64 
- * Ubuntu 17.04, 16.04, 14.04  x64
- * Linux Mint 18, 17 x64
- * openSUSE 42.2+ x64
- * SUSE Enterprise Linux (SLES) 12 SP2+ x64
+ * Red Hat Enterprise Linux 7 x86_64
```

Comment:
The 2.0-support-os.md looks okay, aside from similar concerns about `x64` vs `x86_64` terminlogy.